README.md

```diff
@@ -54,6 +54,109 @@ $ go get -u github.com/skroutz/mistry/cmd/mistry
 
 
 
+Examples
+--------------------------------------------------
+Here is a little preview of how building looks with mistry.
+
+### Bundler
+The project contains a Dockerfile and a docker-entrypoint that describe
+it. We recommend reading the [source](_examples/bundler) of these files as a form of
+documentation for developing other projects.
+
+1. Create the data directory used by the server to store the jobs
+
+```sh
+$ mkdir -p /tmp/mistry/data
+```
+
+2. Create a directory to hold the mistry projects
+
+```sh
+$ mkdir -p /tmp/mistry/projects
+```
+
+3. Copy the example projects to the projects directory
+
+```sh
+$ cp -r _examples/* /tmp/mistry/projects/
+```
```

Comment:
Since these are a one-time setup instructions, they should probably be in a wiki page (ie. "Setup") and just in `examples/README.md` that having completed the "Setup" step is a prerequisite.

Reply:
nice, updated